### PR TITLE
osd: Remvoe backlog entry from pg_log_entry_t

### PR DESCRIPTION
**Pull request by shinobu-x**
Signed-off-by: Shinobu Kinjo <shinobu@redhat.com>

### CODE REVIEW

File: src/osd/osd_types.h [hunk 1]
```diff
@@ -3290,13 +3290,12 @@ struct pg_log_entry_t {
...
     CLONE = 2,    // cloned object from head
     DELETE = 3,   // deleted object
-    BACKLOG = 4,  // event invented by generate_backlog [deprecated]
-    LOST_REVERT = 5, // lost new version, revert to an older version.
-    LOST_DELETE = 6, // lost new version, revert to no object (deleted).
-    LOST_MARK = 7,   // lost new version, now EIO
-    PROMOTE = 8,     // promoted object from another tier
-    CLEAN = 9,       // mark an object clean
-    ERROR = 10,      // write that returned an error
+    LOST_REVERT = 4, // lost new version, revert to an older version.
```
Comment (amitkumar50) [inline src/osd/osd_types.h]: PR heading spelling of remove is wrong

Comment (liewegas) [reply src/osd/osd_types.h]: The IDs can't change as they are part of the on-disk format

**Comment by liewegas** [comment]
needs rebase.. otherwise this looks good!

**Comment by tchaikov** [comment]
closing, as 93ceffaccf871fd42f2c51495ff778bd552082f8 has removed backlog type.

**Pull request closed without merging**